### PR TITLE
docstrings and method rename

### PR DESCRIPTION
**Pull request by AleksanderWWW**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,12 +43,12 @@ from neptune_fetcher import (
...
     - _Method Overriding_:
-        - `series_setup(total_series: int, series_limit: int)`: Sets up a progress bar for series fetching.
+        - `pre_series_fetch(total_series: int, series_limit: int)`: Sets up a progress bar for series fetching.
         - `on_series_fetch(step: int)`: On every step in the series fetching process.
         - `post_series_fetch()`: After series fetching is completed should clean up the resources.
-        - `table_setup()`: Initializes a progress bar for table fetching.
+        - `pre_run_table_fetch()`: Initializes a progress bar for table fetching.
         - `on_run_table_fetch(step: int)`: On every step in the table fetching process.
-        - `post_table_fetch()`: After table fetching is completed should clean up the resources.
+        - `post_run_table_fetch()`: After table fetching is completed should clean up the resources.
```
Comment (normandy7) [inline README.md:51]: Maybe we should use "runs" instead of "run", as we normally only say _runs table_.

Comment (AleksanderWWW) [reply README.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ### Changes
 - Added support for download progress update handling in fetching API ([#25](https://github.com/neptune-ai/neptune-client-experimental/pull/25))
+- - Renamed methods in `ProgressUpdateHandler` ([#26](https://github.com/neptune-ai/neptune-client-experimental/pull/26))
+
```
Comment (Raalsky) [inline CHANGELOG.md:6]:
```suggestion
- Renamed methods in `ProgressUpdateHandler` ([#26](https://github.com/neptune-ai/neptune-client-experimental/pull/26))
```

**Review by Raalsky — APPROVED** [review]

File: src/neptune_fetcher/read_only_project.py [hunk 3]
```diff
@@ -70,14 +70,23 @@ def _get_attribute(entry: TableEntry, name: str) -> Optional[str]:
         return None
 
 
-class FrozenProject:
+class ReadOnlyProject:
     def __init__(
         self,
         project: Optional[str] = None,
         workspace: Optional[str] = None,
```
Comment (normandy7) [inline src/neptune_fetcher/read_only_project.py:77]: Do these work like normal? I.e. you can leave out `workspace` and include the workspce name in the `project` parameter? And it'll fall back to the `NEPTUNE_PROJECT_NAME` env var?

Comment (AleksanderWWW) [reply src/neptune_fetcher/read_only_project.py:77]: Unfotunately, not yet :( It will be the case soon though, but it's a first iteration

Comment (AleksanderWWW) [reply src/neptune_fetcher/read_only_project.py:77]: Api token will work like that though

**Pull request merged**